src/yarp.c

```diff
@@ -249,22 +258,22 @@ yp_node_create_from_token(yp_parser_t *parser, yp_node_type_t type, const yp_tok
 static yp_node_t *
```

Comment:
I think we decided to return `yp_alias_node_t *` here?

Reply:
The reason I'm thinking is there will be lots of places where we create (for example) an arguments node, and then pass that into various functions that append arguments. So you'll get something back for an arguments node that is an `yp_node_t *` and then you'll have to cast it every time it gets passed into one of those functions. Can we try it for the first couple and if it gets really annoying we can change it back?